InAppSettingsKit/Controllers/IASKAppSettingsViewController.m

```diff
@@ -418,6 +418,12 @@ - (void)setSpecifier:(IASKSpecifier*)specifier on:(BOOL)on {
 			[self.settingsStore setBool:NO forSpecifier:specifier];
 		}
 	}
+	NSString* key = specifier.key;
+	if (key != nil) {
```

Comment:
Because of this the setting needs a key, otherwise we cant find the cell. Is there a better way to get the corresponding cell from a specifier?